pkg/orchestrator/etcd_worker.go

```diff
@@ -269,10 +312,22 @@ func (worker *EtcdWorker) applyPatches(ctx context.Context, patches []DataPatch)
 			op = clientv3.OpPut(key.String(), string(value))
 		} else {
 			op = clientv3.OpDelete(key.String())
+			hasDelete = true
 		}
 		ops = append(ops, op)
 	}
 
+	if hasDelete {
+		ops = append(ops, clientv3.OpPut(worker.prefix.String()+deletionCounterKey, fmt.Sprint(worker.deleteCounter+1)))
```

Comment:
Does this key exist in etcd permanently and always increase, do we need any gc mechanism

Reply:
Yes, it will always increase and it is not expected to be deleted unless the CDC cluster is reset. Developing a GC mechanism is unnecessary because the benefit is small.